### PR TITLE
Challenge 4 README - Deposit and Withdraw functions extra info, and modify Ship your contracts section

### PR DESCRIPTION
**Pull request by mertcanciy**
Add extra explanations in README to `deposit()` and `withdraw()` functions, discussed in #58, and modify `Ship your contracts` section to delete screenshots and add `fromBlock` passed to `useScaffoldEventHistory` hint.

Complete list of changes:

- Remove side quest: `Side Quest: Prove your knowledge by calculating the exact amount of Balloons you need to approve when depositing. Don't forget about the fee!` (previously added in this PR)
- Explain in the Readme to add a high enough quantity to not have problems with allowance.
- Change description of `withdraw` like it is in last @Pabl0cks comment
  > 💬💬 More Hints: The withdraw() function lets a user take his Liquidity Provider Tokens out, withdrawing both ETH and $BAL tokens out at the correct ratio.....
- Modify  `Ship your contracts` section, using generic content without screenshots.
- Add  `fromBlock` passed to `useScaffoldEventHistory` hint.

### CODE REVIEW

**Comment by Pabl0cks** [comment]
I've just pushed the changes from [rinat summary](https://github.com/scaffold-eth/se-2-challenges/pull/58#issuecomment-1677698055)

Also cleaned **Deploy your contracts** section to delete chal0 images, mentioned [here](https://github.com/scaffold-eth/se-2-challenges/issues/57#issuecomment-1677268918).

Got the content without screenshots from Chal 5 and added the "`fromBlock` passed to `useScaffoldEventHistory`" Hint.

**Comment by Pabl0cks** [comment]
Should we update PR title and description to be accurate with the final changes we're pushing here?

Something like this for the title: Chal, 4 Readme - Deposit and WIthdraw explanations and delete Ship your contracts screenshots

File: README.md [hunk 1]
```diff
@@ -357,39 +359,25 @@ Cool beans! Your front-end should be showing something like this now!
...
-
-🚀 Deploy your BAL and DEX smart contracts:
-
-```shell
-yarn deploy
-```
+🚀 Run `yarn deploy` to deploy your smart contracts to a public network (selected in `hardhat.config.ts`)
 
-> 💬 Hint: You can set the `defaultNetwork` in `hardhat.config.ts` to `sepolia` **OR** you can `yarn deploy --network sepolia`.
+> 💬 Hint: For faster loading of your _"Events"_ page, consider updating the `fromBlock` passed to `useScaffoldEventHistory` in [`packages/nextjs/pages/events.tsx`](https://github.com/scaffold-eth/se-2-challenges/blob/challenge-2-token-vendor/packages/nextjs/pages/events.tsx) to `blocknumber - 10` at which your contract was deployed. Example: `fromBlock: 3750241`.
```
Comment (rin-st) [inline README.md:380]: Link to challenge 2

**Comment by rin-st** [comment]
> Should we update PR title and description to be accurate with the final changes we're pushing here?

Yes, update it please

File: README.md (hunk 1 above)
Comment (Pabl0cks) [reply README.md]: Fixed, sorry! Nice catch 🙌

**Comment by Pabl0cks** [comment]
> Yes, update it please

Title and description updated 👌

**Comment by ZakGriffith** [comment]
This all looks good, nice work.  I would be fine with a squash and merge, then we can step back and take another pass at it.

**Review by ZakGriffith — APPROVED** [review]

**Review by ZakGriffith — APPROVED** [review]

**Comment by Pabl0cks** [comment]
> This all looks good, nice work. I would be fine with a squash and merge, then we can step back and take another pass at it.

Done! 🙌 Had to resolve a small conflict in the **Deploy your smart contracts** section, kept your last changes, adding the extra Hint `fromBlock` passed to `useScaffoldEventHistory`.

**Pull request merged**